src/content/docs/style-guide/word-choice/nerdgraph-doc-guidelines.mdx

```diff
@@ -19,15 +19,22 @@ Some style-related guidelines when writing about NerdGraph:
+If you're adding content to the NerdGraph docs category, consider the following tips for helping keep our content efficiently organized and easy to find:
+* **Consider adding content to an existing tutorial.** When adding new NerdGraph content, check to see if you can include new content in existing NerdGraph docs. This helps us keep the list of tutorials clean and prevent it from sprawling. For example, if you want to add something related to viewing or configuring entities, you might add that to the [entity-related NerdGraph doc](/docs/apis/nerdgraph/examples/nerdgraph-entities-api-tutorial).
+* **Choose a doc to emulate.** When writing a new NerdGraph tutorial doc, look through the existing tutorial docs and choose one that is most similar to your content. Copy that and use it as a template. 
 * When you add a new type of NerdGraph content, add it to the [index of NerdGraph content](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#tutorials). That index was created to reflect that we have tutorials available that aren't in the main NerdGraph docs section.
-* To better format requests: If you want to format a request in NerdGraph explorer, click **Tools > Prettify**. The request must be a working one, with real values.
+
+Some tips when writing NerdGraph content: 
+* **Example requests.** Tips for creating example requests: 
+  * Use `<var>` tags for customer-specific values in example queries and mutations. Remember that you don't need to use `<var>` tags in example responses or error messages. 
+  * It's good practice to explain the general use of a request, followed by a note that the example request is just an example (as opposed to communicating that it's the only way to accomplish something). So for example, you might say, "You can use NerdGraph to update your [API test synthetic monitor](/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors#types-monitors). Here's an example of creating an API test monitor that...". 
+  * To clean up the format of a request, go into NerdGraph explorer and input the request and click **Tools > Prettify**. Note that the request must be a working one, with real values.
+* **Field definitions.** The NerdGraph explorer includes built-in field definitions, but it can be good to explain any of the more complex or non-obvious fields in your doc. For common fields used in all or most of the example requests, one way to tackle this is with an "Important concepts" section near the top of the doc, in which you include any recommended reading or notes on important fields. For fields that are only used in one or two requests, you may prefer to explain them when introducing the example request. 
```

Comment:
I poked around our NerdGraph docs to see. Do we preface NerdGraph explorer with an article each time? I was wondering if the style guide for writing about NerdGraph included stuff like that.